src/dapp/libexec/dapp/dapp-mk-standard-json

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import json
+import subprocess
+import os
+import re
+import fnmatch
+from pathlib import Path
+
+template = open(Path(__file__).parent / "../../resources/jsontemplate.json").read()
+
+# Remove comments from template
+tmpljson = json.loads(re.sub(r'//.*\n', '', template))
```

Comment:
`re.sub(r'^\s*//.*\n', ...)` to avoid matching strings that happen to contain "//"